.github/workflows/ci-test.yml

```diff
@@ -0,0 +1,41 @@
+name: CI Tests
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [ master ]
+  merge_group:
+    types: [ checks_requested ]
+
+jobs:
+  ci:
+    name: pester
+    strategy:
+      matrix:
+        os: [ windows-latest, macos-latest, ubuntu-latest ]
+    runs-on: ${{ matrix.os }}
+    env:
+      DOTNET_NOLOGO: true
+      DOTNET_CLI_TELEMETRY_OPTOUT: true
+      DOTNET_GENERATE_ASPNET_CERTIFICATE: false
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-dotnet@v4
+        with:
+          dotnet-version: 8.x
+      - run: ./tools/installPSResources.ps1
+        shell: pwsh
+      - run: ./build.ps1 -Build && Invoke-ModuleTests -Type Functional
```

Comment:
AFAICT "Functional" is the _only_ test type that exists anyway, so it's superfluous but is what the ADO build was doing (and is the default if not included like this anyway).